### PR TITLE
Translated admin menu titles

### PR DESCRIPTION
**Pull request by yogeshsuhagiya**
### Description
Added `translate="title"` for admin menus
Removed extra spaces from `<add>` tags

### Fixed Issues (if relevant)

1. N/A

### Manual testing scenarios

1. N/A

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @Yogeshks. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation

**Comment by sidolov** [comment]
@magento-engcom-team combine 17525

**Comment by magento-engcom-team** [comment]
Hi @sidolov. Thank you for your request. I'm working on combining the pull requests for you

**Comment by magento-engcom-team** [comment]
@Yogeshks all pull requests have been successfully combined together: 
 - merged https://github.com/magento/magento2/pull/17525:  Translated acl rule title

File: app/code/Magento/Braintree/etc/adminhtml/menu.xml [hunk 1]
```diff
@@ -7,13 +7,6 @@
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Backend:etc/menu.xsd">
     <menu>
-        <add
```
Comment (VladimirZaets) [inline app/code/Magento/Braintree/etc/adminhtml/menu.xml]: Hi @Yogeshks. What the reason to move attributes to one line? As for me, the previous format is more readable. Also, Magento has the rules that check line length in the code.

Comment (yogeshsuhagiya) [reply app/code/Magento/Braintree/etc/adminhtml/menu.xml]: Hi @VladimirZaets, You're right but I had reviewed all `menu.xml` files inside `app/code/Magento` directory and found that `<add>` is written in one line inside all files. This is the only reason.

Comment (yogeshsuhagiya) [reply app/code/Magento/Braintree/etc/adminhtml/menu.xml]: @VladimirZaets, Please let me know if you still need updates.

Comment (yogeshsuhagiya) [reply app/code/Magento/Braintree/etc/adminhtml/menu.xml]: Hi @VladimirZaets, Please let me know, need to update it?

**Comment by yogeshsuhagiya** [comment]
@magento-engcom-team @VladimirZaets Hi, any updates for this PR?

**Comment by magento-engcom-team** [comment]
Hi @Yogeshks. Thank you for your contribution. 
We will aim to release these changes as part of 2.2.7. 
Please check the release notes for final confirmation.

Please, consider to port this solution to 2.3 release line.
You may use [Porting tool](https://github.com/magento/magento2/wiki/Simple-way-to-port-your-solutions-automatically) to port commits automatically.

**Pull request merged**